readme.md

```diff
@@ -104,6 +104,10 @@ Check if an item exists.
 
 Delete an item.
 
+#### .onDidChange([key], callback)
+
+Watches the given `key`, calling `callback` on any changes. If `key` is not given, then `callback` will be called after any change is made to the store.
```

Comment:
We need to document what parameters `callback` provides. And that `oldValue` is `undefined` when a key is first set and `newValue` is `undefined` if a key is deleted.

What should the `newValue`/`oldValue` be when the user doesn't specify a `key`? I guess the store itself, but is that really useful? I'm trying to think of actual use-cases for this feature? What does Atom return in that case, for example.

Reply:
[Atom returns the store itself](https://github.com/atom/key-path-helpers/blob/master/lib/key-path-helpers.js#L17). I personally don't think I'd ever use this to watch the whole store though.